File: A-SOURCE_CODE/A-SERVICE/Shared/ViewModels/Categories/CategoryViewModel.cs
using Newtonsoft.Json;
using Shared.ViewModels.Accounts;

namespace Shared.ViewModels.Categories
{
    public class CategoryViewModel
    {
        /// <summary>
        ///     Id of category
        /// </summary>
        [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
        public int? Id { get; set; }

        /// <summary>
        ///     Person who created the categories.
        /// </summary>
        [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
        public AccountViewModel Creator { get; set; }

        /// <summary>
        ///     Name of category.
        /// </summary>
        [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
        public string Name { get; set; }

        /// <summary>
        ///     When the category created.
        /// </summary>
        [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
        public double? Created { get; set; }

        /// <summary>
        ///     When the category was lastly modified.
        /// </summary>
        [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
        public double? LastModified { get; set; }
    }
}

File: A-SOURCE_CODE/B-CLIENT/A-WEB/Administration/src/viewmodels/category/category.view-model.ts
import {Account} from '../../models/entities/Account'
export class CategoryViewModel{

  //#region Properties

  /*
  * Category index.
  * */
  public id: number;

  /*
  * Who created the category.
  * */
  public creator: Account;

  /*
  * Category name.
  * */
  public name: string;

  /*
  * Time when category was created.
  * */
  public createdTime: number;

  /*
  * Time when category was lastly modified.
  * */
  public lastModifiedTime: number;

  //#endregion

}
